core-drawer-panel.html

```diff
@@ -75,344 +75,425 @@
 @homepage github.io
 -->
 
+<link rel="import" href="../polymer/polymer.html">
+
 <link rel="import" href="../core-media-query/core-media-query.html">
 <link rel="import" href="../core-selector/core-selector.html">
 
-<polymer-element name="core-drawer-panel" touch-action="auto">
-<template>
-
+<dom-module id="core-drawer-panel">
   <link rel="stylesheet" href="core-drawer-panel.css">
 
-  <core-media-query query="max-width: {{forceNarrow ? '' : responsiveWidth}}" queryMatches="{{queryMatches}}"></core-media-query>
+  <template>
+    <core-media-query query="[[mediaQuery]]" on-query-matches-changed="onQueryMatchesChanged"></core-media-query>
+
+    <core-selector class-name="[[coreSelectorClass]]" attrForSelected="id" selected="{{selected}}">
```

Comment:
I'd be happy to ditch it, but this is the type of case where the core team seemed to prefer it.